web/src/routes/admin/repair/+page.svelte

```diff
@@ -137,12 +140,15 @@
         count += await loadAndMatch(filenames.slice(index, index + chunkSize));
       }
     } catch (error) {
-      handleError(error, $t('errors.unable_to_check_items'));
+      handleError(error, $t('errors.repair_unable_to_check_items'));
```

Comment:
I feel like this should be done with plurals but we don't have the count here :/

Reply:
Plurals don't seem right here, I've used the [{select} format](https://formatjs.io/docs/core-concepts/icu-syntax/#select-format) instead

Reply:
The `one` option can be omitted if there's no difference for a language

Reply:
Oh awesome!